content/guides/content-management/permission-editor.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Permission editor"
+metaTitle: "sensenet admin-ui - Permission editor"
+metaDescription: "sensenet admin-ui - Permission editor"
+---
+# Permission editor
+
+sensenet offers a sophisticated permission system, giving you the possibility to apply permission rules according to the logic you prefer. The permission editor is available on the admin ui.
+
+There are a lot of built-in permissions but there are some reserved ones that you are free to use in case you have special needs (custom permissions). You can also group these permissions for the permission editor. The related settings file is available under Setup menu / Permission.settings.
+On the permission editor dialog you can see the groups and permissions based on these settings.
+
+You can assign users to roles taking advantage of predefined permission settings to that role. Since permissions are not directly assigned to users, they only acquire them through their role (or roles), management of individual user rights becomes a matter of simply assigning appropriate roles to specific users; this simplifies common operations, such as adding a new user, or changing a user’s department.
```

Comment:
```suggestion
#Permission inheritance
One of the advantages of the tree based hierarchical content structure in sensenet, is that it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. You can assign users to roles taking advantage of predefined permission settings to that role. Since permissions are not directly assigned to users, they only acquire them through their role (or roles), management of individual user rights becomes a matter of simply assigning appropriate roles to specific users; this simplifies common operations, such as adding a new user, or changing a user’s department.

If you set a permission on a document library it will be automatically applied to subfolders and contents. Inherited permissions cannot be changed without breaking the inheritance. To learn more about breaking the inheritance go to [Role based permissions(concept docs)](/concepts/user-and-permission-management/03-role-based-permissions.md).
```